src/app/components/auth/IsAuthorizedWrapper/IsAuthorizedWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import { Spinner } from "@amsterdam/asc-ui"
+import { usePermissions } from "app/state/rest"
+
+/* 
+ ** IsAuthorizedWrapper is used to manage permissions.
+ ** When user has the appropriate permission, the children will be returned.
+ ** When the permission cannot be found or the permission is false, null will be returned.
+ */
+
+type Props = {
+  permName: keyof MockComponents.Schemas.Permissions["permissions"]
```

Comment:
Better use the full word `permissionName` instead of `permName`. Easier to read and to remember when you have to pass that property.